vms/platformvm/state/state.go

```
@@ -1545,12 +1559,12 @@ func (s *state) loadCurrentValidators() error {
 				return err
 			}
 
-			stakerTx, ok := tx.Unsigned.(txs.Staker)
-			if !ok {
-				return fmt.Errorf("expected tx type txs.Staker but got %T", tx.Unsigned)
-			}
```

just moved up a bit. Should never happen but always better to fail fast